shenyu-spring-boot-starter/shenyu-spring-boot-starter-sdk/src/main/java/org/apache/shenyu/springboot/starter/sdk/ShenyuSdkAutoConfiguration.java

```diff
@@ -69,16 +70,18 @@ public Contract springMvcContract() {
      * okHttpShenyuSdkClient.
      *
      * @param config config
+     * @param requestInterceptors the request interceptors
      * @param instanceRegisterRepository the instance register repository
      * @return {@link ShenyuSdkClient}
      */
     @Bean
     public ShenyuSdkClient shenyuSdkClient(final RegisterConfig config,
+                                           @Autowired(required = false) final List<ShenyuSdkRequestInterceptor> requestInterceptors,
```

Comment:
maybe used ObjectProvider<List<ShenyuSdkRequestInterceptor>> requestInterceptors ?

Reply:
It makes sense